src/hooks/useSelectionLogic.ts

```diff
@@ -128,7 +129,7 @@ export function useSelectionLogic<T extends HTMLElement>({
           }
           containerRef.current?.drawSelectionBox(newSelectionBox);
           currentSelectionChange.current?.(boxInContainer);
-        } else if (isSelecting) {
+        } else if (isSelecting.current) {
```

Comment:
@tomaszczura this is the actual bug fix. Took the fix from [@yoole5699's PR](https://github.com/AirLabsTeam/react-drag-to-select/pull/34/files#diff-1fa8fd0fedac2a98b4ebd3ef62cab7e2de5b963eb6472c533b6c4d3c69c1621eL134) and added an eslint rule to prevent this from happening again